packages/editor/package.json

```diff
@@ -34,8 +34,10 @@
     "@chakra-ui/icons": "^1.0.15",
     "@chakra-ui/react": "^1.7.1",
     "@emotion/css": "^11.7.1",
+    "@sinclair/typebox": "^0.21.2",
     "@sunmao-ui/chakra-ui-lib": "^0.2.0",
     "@sunmao-ui/core": "^0.4.0",
+    "@sunmao-ui/editor-sdk": "^0.5.0-alpha1",
```

Comment:
Did `editor-sdk` publish this version?

Reply:
Has not yet been, but the `lerna` would parse it as a local package. What version of `editor-sdk` should use here?

Reply:
0.0.0 for now.